### DISCUSSION TITLE
上传大于3m图片就显示    服务端异常，请稍后重试

### DISCUSSION THREAD

**Discussion by pengwangzai** [lsky-org/lsky-pro#609] [问答]
上传大于3m图片就出现情况 后台也会有图片就是前台不显示链接返回的是 , 服务端异常，请稍后重试
![屏幕截图 2023-02-03 232350](https://user-images.githubusercontent.com/96305626/216641113-eddef5d4-01f0-4230-b814-0149b14057ed.png)

**Comment by 0xxb** [comment]
![image](https://user-images.githubusercontent.com/22728201/216739302-95b87233-90bd-4503-9464-fc130d0c0fec.png)

**Comment by abloom25** [comment]
https://github.com/lsky-org/lsky-pro/discussions/357
可以看看这个，我用这个方法安装后就没问题
